assets/css/mobile/pages/sunglasses.css

```diff
@@ -25,3 +25,42 @@ img.isInCartMark, img.loadingIcon {
     text-align: center;
     margin-bottom: .5em;
 }
+
+#mobile-shop-benefits-infographic {
+	position: relative;
+	width: 100%;
+}
+
+.benefits-container {
+	position: relative;
+	margin-left: auto;
+	margin-right: auto;
+	width: auto;
+	max-width: 750px;
+	text-align: center;
+}
+
+.mobile-shop-benefits-infographic-icon {
+	position: relative;
+	display: inline-block;
+}
+
+@media all and (max-width: 410px) {
+	.mobile-shop-benefits-infographic-icon {
+		width: 50%;
+	}
+}
+
+@media (min-width: 411px) and (max-width: 650px) {
```

Comment:
Quite complicated css with two breakpoints (`410px`, `650px`) and three width (`50%`, `33%`, `19%`).
It's ok, but I've got a feeling that almost the same styling could be done in a simpler and more elegant way, which would be less error-prone and easier to support / modify.
https://en.wikipedia.org/wiki/KISS_principle